src/NosCore.Controllers/ExchangePacketController.cs

```diff
@@ -260,20 +287,23 @@ public void RequestExchange(ExchangeRequestPacket packet)
                         Session.Character.AddBankGold(getTargetData.BankGold * 1000);
                     }
 
-                    closeExchange = _exchangeService.CloseExchange(Session.Character.VisualId, success ? ExchangeCloseType.Success : ExchangeCloseType.Failure);
+                    closeExchange = _exchangeService.CloseExchange(Session.Character.VisualId, success.Item1 ? ExchangeCloseType.Success : ExchangeCloseType.Failure);
```

Comment:
or you can put success.Item1 as a ExchangeCloseType
also i would rename ExchangeCloseType in ExchangeResultType